src/paint/supports.c

```diff
@@ -21,15 +21,15 @@
 
 /** rct2: 0x0097AF20, 0x0097AF21 */
 const rct_xy8 loc_97AF20[] = {
-	{4,  4},
-	{28, 4},
-	{4,  28},
-	{28, 28},
-	{16, 16},
-	{16, 4},
-	{4,  16},
-	{28, 16},
-	{16, 28}
+	{ { {4,  4} } },
+	{ { {28, 4} } },
+	{ { {4,  28} } },
+	{ { {28, 28} } },
+	{ { {16, 16} } },
+	{ { {16, 4} } },
+	{ { {4,  16} } },
+	{ { {28, 16} } },
+	{ { {16, 28} } }
```

Comment:
Can't we wrap these in some kind of `MakeXY8` macro instead of three layers of braces?